jobs/v4/src/main/java/com/example/jobs/JobSearchCreateJob.java

```diff
@@ -46,14 +48,27 @@ public static void createJob(
       String requisitionId,
       String jobApplicationUrl)
       throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
     try (JobServiceClient jobServiceClient = JobServiceClient.create()) {
       TenantName parent = TenantName.of(projectId, tenantId);
       Job.ApplicationInfo applicationInfo =
           Job.ApplicationInfo.newBuilder().addUris(jobApplicationUrl).build();
 
-      List<String> addresses = Arrays.asList("1600 Amphitheatre Parkway, Mountain View, CA 94043",
+      List<String> addresses =
+          Arrays.asList(
+              "1600 Amphitheatre Parkway, Mountain View, CA 94043",
               "111 8th Avenue, New York, NY 10011");
 
+      // if exp date is not set, by default it will expire in 30 days.
+      Timestamp expirationDate =
+          Timestamps.add(
+              Timestamps.fromMillis(System.currentTimeMillis()),
+              com.google.protobuf.Duration.newBuilder()
+                  .setSeconds(60L * 60L * 24L * 30L) //  30 days ago
+                  .build());
```

Comment:
https://docs.oracle.com/javase/8/docs/api/java/sql/Timestamp.html <== Is protobuf timestamp leaked to users?  Is this really a good idea?

Reply:
I may have copied the wrong one.  I think LocalDate https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html is the Java 8 hotness. (if it works)

Reply:
And LocalDateTime https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html

Reply:
I referenced from appengine Java 8 
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/caebd85cdda8e3647e33f9f83c64ea3b1b0a0673/appengine-java8/bigquery/src/main/java/com/example/appengine/bigquerylogging/BigQueryRunner.java#L199